From e92bd1aee5b2d5692244f040d3d34f9f3d4ea022 Mon Sep 17 00:00:00 2001
From: Michael Shamis <michaelsh@marvell.com>
Date: Fri, 15 Jan 2021 21:39:45 +0200
Subject: [PATCH 1155/1921] dts: marvell: 913x-db: enable crypto

Enable SAM driver for 913x-db platforms.

Signed-off-by: Michael Shamis <michaelsh@marvell.com>
Change-Id: I44bccd38dcb91a61dbfde85562d2d235cdfa68ab
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/43910
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-db.dts | 4 ----
 arch/arm64/boot/dts/marvell/cn9131-db.dts | 4 ----
 arch/arm64/boot/dts/marvell/cn9132-db.dts | 4 ----
 3 files changed, 12 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-db.dts b/arch/arm64/boot/dts/marvell/cn9130-db.dts
index f03f93426b6d..420b099fd468 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-db.dts
+++ b/arch/arm64/boot/dts/marvell/cn9130-db.dts
@@ -117,10 +117,6 @@
 	status = "okay";
 };
 
-&cp0_crypto {
-	status = "disabled";
-};
-
 &cp0_ethernet {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/marvell/cn9131-db.dts b/arch/arm64/boot/dts/marvell/cn9131-db.dts
index 408786cf8236..920b68cfb340 100644
--- a/arch/arm64/boot/dts/marvell/cn9131-db.dts
+++ b/arch/arm64/boot/dts/marvell/cn9131-db.dts
@@ -77,10 +77,6 @@
 #undef CP11X_PCIE1_BASE
 #undef CP11X_PCIE2_BASE
 
-&cp1_crypto {
-	status = "disabled";
-};
-
 &cp1_ethernet {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/marvell/cn9132-db.dts b/arch/arm64/boot/dts/marvell/cn9132-db.dts
index c3124d849e3a..c1b635427541 100644
--- a/arch/arm64/boot/dts/marvell/cn9132-db.dts
+++ b/arch/arm64/boot/dts/marvell/cn9132-db.dts
@@ -95,10 +95,6 @@
 #undef CP11X_PCIE1_BASE
 #undef CP11X_PCIE2_BASE
 
-&cp2_crypto {
-	status = "disabled";
-};
-
 &cp2_ethernet {
 	status = "okay";
 };
-- 
2.31.1

